.github/workflows/deploy-zeit-production.yml

```diff
@@ -58,14 +75,23 @@ jobs:
     steps:
       - uses: actions/checkout@v1 # Get last commit pushed - XXX See https://github.com/actions/checkout
       - name: Resolving deployment url from Zeit
-        # The following workflow is:
-        #  - getting all deployments data (by using the scope in `now.json`)
-        #  - then we get the last url (in Node.js it corresponds as `response.deployments[0].url`
-        #  - and then we remove the `"` character to pre-format url
-        # We need to set env the url for next step, formatted as `https://${url provided by API}`
+        # Workflow overview:
+        #   - Resolve customer to deploy from github event input (falls back to resolving it from now.json file)
+        #   - Resolve $ZEIT_DEPLOYMENT
+        #     - Fetch all deployments data (by using the scope in `now.json`)
+        #     - Resolve the last url (from `response.deployments[0].url`)
+        #     - Remove the `"` character to pre-format url
+        # We need to set env the url for next step, formatted as `https://${$ZEIT_DEPLOYMENT}`
         run: |
           apt update -y >/dev/null && apt install -y jq >/dev/null
-          ZEIT_DEPLOYMENT=`curl -H 'Accept: application/json' -H 'Content-Type: application/json' -H 'Authorization: Bearer ${{ secrets.ZEIT_TOKEN }}' https://api.zeit.co/v5/now/deployments?teamId=$(cat now.json | jq -r '.scope') | jq '.deployments [0].url' | tr -d \"`
+
+          MANUAL_TRIGGER_CUSTOMER="${{ github.event.inputs.customer}}"
+          echo "MANUAL_TRIGGER_CUSTOMER: " $MANUAL_TRIGGER_CUSTOMER
+
+          CUSTOMER_TO_DEPLOY="${MANUAL_TRIGGER_CUSTOMER:-$(cat now.json | jq -r '.build.env.NEXT_PUBLIC_CUSTOMER_REF')}"
+          echo "Customer to deploy: " $CUSTOMER_TO_DEPLOY
+
+          ZEIT_DEPLOYMENT=`curl -H 'Accept: application/json' -H 'Content-Type: application/json' -H 'Authorization: Bearer ${{ secrets.ZEIT_TOKEN }}' https://api.zeit.co/v5/now/deployments?teamId=$(cat now.$CUSTOMER_TO_DEPLOY.production.json | jq -r '.scope') | jq '.deployments [0].url' | tr -d \"`
```

Comment:
@Demmonius This is too complicated to my taste, will be very hard to debug when something will go wrong.

Could it be possible to split it under a few variables, `echo` those variables and only then run the command? Basically reduce complexity by splitting it.

This would tremendously help debug by printing the various parts/variables and would reduce the overall debug complexity.